### ISSUE TITLE
Empty logic

### ISSUE THREAD

**Issue by danieldanielecki** [JamesSkemp/fishy-fishy-phaser-game#1]
This game looks quite promising based on the assets which are supposed to be used, however found that there's not much logic in that game :(

**Comment by JamesSkemp** [comment]
Yeah, this was something I was going to make for my son, since he loved fish, but I never ended up with the time to build it out.

I've hit a mental block with the game I was making for myself so ... Maybe I'll switch back to this one. Didn't need much time to do what I wanted it to do. :)
